script/foreman-debug

```diff
@@ -125,6 +125,9 @@ add_files() {
       add_file $FILE
       SUMSIZE=$(($((SUMSIZE))+$((SIZE))))
     fi
+    touch -c -r "$FILE"   "$DIR$FILE"
+    touch -c -r "$FILE"   "$DIR$FILE.txt"
+    touch -c -r "$SUBDIR" "$DIR$SUBDIR"
```

Comment:
For directory related one, if we set the timestamp right after creating them with mkdir, the timestamp (mtime) would be overwritten every time a new file/directory is created within it. Which is why I thought of adding `touch -c -r "$SUBDIR" "$DIR$SUBDIR"` at the end of `for` loop in `add_files()`.

I originally thought of not adding multiple lines of the same command and adding it at the end of the loop. Will move the file related touch commands to immediately after they're created.